docker/docker-bake.hcl

```diff
@@ -13,31 +13,35 @@ variable "IMAGE" {
 variable "VERSION" {
   default = "unknown"
 }
-variable "PLATFORM" {
-  default = "linux/amd64"
+variable "PLATFORMS" {
+  default = "linux/amd64,linux/arm64"
+}
+variable PLATFORM_LIST {
+  default = split(",", PLATFORMS)
 }
 variable "BRANCH" {
   default = "stable"
 }
-
-function "get_platform_tag" {
-  params = []
-  result = replace("${equal(PLATFORM, "") ? "" : "-"}${PLATFORM}", "linux/", "")
+variable "ECR_REPO" {
+  default = "454861456664.dkr.ecr.us-east-2.amazonaws.com"
```

Comment:
Is it ok to publish this info in this repository?

Reply:
NVM, it was already in the code.

Reply:
Good catch but yeah, I checked :)